app/views/event_attendances/index.html.erb

```diff
@@ -1,91 +1,111 @@
 <h3 class="pre-title"><%= @event.name %></h3>
 <h1><%=h @audience.to_s %> Attendance</h1>
-	
+
 <table class="attendees">
-	<thead>
```

Comment:
Yes. I will get back home and do it today.

Kritin Vij
University of Washington
Computer Science | Math Minor
*kritinvij@gmail.com <kritinvij@gmail.com> | *
<http://in.linkedin.com/pub/kritin-vij/2b/176/ab0/>

On Tue, Jun 23, 2015 at 11:57 PM, Mitchell Harper <notifications@github.com>
wrote:

> In app/views/event_attendances/index.html.erb
> <https://github.com/uwdreamproject/DreamSIS/pull/145#discussion_r33121558>
> :
>
> >  <table class="attendees">
> > -	<thead>
>
> @kritinvij <https://github.com/kritinvij> Could you please go through
> these lines and change the spaces back to tabs, to keep the diff a little
> smaller?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uwdreamproject/DreamSIS/pull/145/files#r33121558>.
>